McCorrections/include/LeptonWeights.h

```diff
@@ -145,23 +145,31 @@ class LeptonWeights : public IWeightProvider {
                   const std::string& muon_SingletriggerInput, const std::string& muon_CrossTriggerInput,
                   const std::string& tauTriggerInput, Period period, DiscriminatorWP _tau_iso_wp);
```

Comment:
add boolean to indicate whatever to use dm or pt dependent SFs for true taus and use it later in the code.